src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-form/rgw-user-form.component.html

```diff
@@ -77,6 +77,9 @@ <h3 class="panel-title">{editing, select, 1 {Edit} other {Add}} user</h3>
             <span class="help-block"
                   *ngIf="userForm.showError('email', frm, 'email')"
                   i18n>This is not a valid email address.</span>
+            <span class="help-block"
+                  *ngIf="userForm.showError('email', frm, 'notUnique')"
+                  i18n>The chosen email ID is already in use.</span>
```

Comment:
How about this, to be in line with nomenclature used above?
```suggestion
                  i18n>The chosen email address is already in use.</span>
```

Reply:
ack